api/app/src/routes/helpers/provider-route-helper.ts

```diff
@@ -46,7 +46,7 @@ export async function getProviderDataForType<T>(
     } else {
       console.error(result.reason);
       capture.error(result.reason, {
-        extra: { context: `${providers[i]}.${type}` },
+        extra: { context: `getProviderDataForType`, additional: `${providers[i]}.${type}` },
```

Comment:
Couldn't find the `fitbit.Activity` text on code from this issue's additional/extra context: https://metriport-inc.sentry.io/issues/4246508388/events/4e32f46c6e784b769f3cd324f220be3c/